tenpy/linalg/symmetries.py

```diff
@@ -728,22 +721,51 @@ def factor_where(self, descriptive_name: str) -> int:
                 return i
         raise ValueError(f'Name not found: {descriptive_name}')
 
-    def qdim(self, a: Sector) -> int:
+    def qdim(self, a: Sector) -> float:
         if self.is_abelian:
             return 1
 
         dims = []
         for i, f_i in enumerate(self.factors):
             a_i = a[self.sector_slices[i]:self.sector_slices[i + 1]]
             dims.append(f_i.qdim(a_i))
-        return np.prod(dims)
+        #print('symmetry',type(np.prod(dims)))
```

Comment:
please remove random comments like this.
I suggest using the python debugger instead of putting print statements.